Unit tests process crashes

#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
The unit tests should work on master branch.

#### What is the current behavior?
The process fails in step `:build:components:spec` with:
```
node_modules/@types/jasmine/index.d.ts(39,38): error TS2371: A parameter initializer is only allowed in a function or constructor implementation.
node_modules/@types/jasmine/index.d.ts(39,46): error TS2304: Cannot find name 'keyof'.
node_modules/@types/jasmine/index.d.ts(39,52): error TS1005: '=' expected.
```

#### What are the steps to reproduce?
Run `gulp test` on the master branch.

#### Which versions of Angular, Material, OS, browsers are affected?
Latest Version of Material, Windows 10, Angular 2.4.7

#### Is there anything else we should know?
I'm no expert in unit tests, but simply starting them on a new cloned repository should work (e2e tests are working fine). It could be possible too, that this is just a problem on my system, because otherwise someone would have already noticed this.

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
     "@types/glob": "^5.0.30",
     "@types/gulp": "^3.8.32",
     "@types/hammerjs": "^2.0.34",
-    "@types/jasmine": "^2.5.41",
+    "@types/jasmine": "2.5.41",
     "@types/merge2": "^0.3.29",
     "@types/minimist": "^1.2.0",
     "@types/node": "^7.0.4",

